advocacy_docs/partner_docs/VeritasGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring NetBackup for PostgreSQL'
+---
+
+
+### Prerequisites
+
+- A running EDB Postgres Advanced (EPAS)
+- A running NetBackup Server with a DataStore Policy set up
+- NetBackup Client installed
```

Comment:
where is agent installed?  On all EPAS?  just one?

Reply:
Both client and agent installed are on the EPAS host